package.json

```diff
@@ -96,9 +96,11 @@
     "any-path": "^1.3.0",
     "chai": "^4.2.0",
     "coveralls": "^3.0.3",
+    "cross-env": "^5.2.0",
     "is-windows": "^1.0.2",
     "lodash": "^4.17.11",
     "newline-regex": "^0.2.1",
+    "pify": "^4.0.1",
```

Comment:
@bcoe we're ok with adding this dependency? as @coreyfarrell said `execa` is already brought in and would be great to use here.

Reply:
This is temporary, can be replaced with `util.promisify` once we drop node.js 6.